website/docs/reference/integrations/jira-cloud-plugin-installation.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Jira Cloud Integration - Installation
+---
+
+## Prerequisites
+
+### Jira Cloud
+
+- [Jira **Cloud**](https://www.atlassian.com/software/jira)
+
+For Jira Data Center, check out the [Jira Server plugin](jira-server-plugin-installation.md)
+
+### Unleash
+
+- Unleash v4 or higher
+
+## Required access levels
+
+### Unleash
+
+You will need an Unleash admin user to configure the access tokens needed to connect the plugin to Unleash.
+
+For Unleash Open Source this plugin requires the use of an [Admin API token](../api-tokens-and-client-keys.mdx) to connect.
+
+For Unleash Enterprise we recommend using a [service account](../service-accounts.md). Service accounts are also required to integrate with [Change Requests](../change-requests.md)
+
+### Jira
+
+You will need a Jira admin user.
+
+## Installation
+
+If you have an Unleash enterprise license you're welcome to reach out to us at sales@getunleash.io for a free plugin license, otherwise you'll need to try the plugin for 30 days free or purchase a license through the marketplace.
+
+> More to be added here after we can have screenshots and guides of the installation procedure
```

Comment:
It was intended, but now that you mention it it does feel a bit `// TODO:` and can be taken care of in a better way process wise. will fix

Reply:
I vote we remove this. We do need the review to complete but that's uncertain since it's on Atlassian timelines. I'd rather have our docs in a nice state than have confusing looking placeholders

Reply:
Removed!